### ISSUE TITLE
installation fail: no glibtoolize or libtoolize

### ISSUE THREAD

**Issue by dorugeber** [firedrakeproject/firedrake#780]
Last few entries of the log are

```
2016-05-13 18:34:55,017 INFO   Installing libspatialindex

2016-05-13 18:34:55,017 INFO   Cloning libspatialindex

2016-05-13 18:34:56,996 DEBUG  
2016-05-13 18:34:56,996 INFO   Successfully cloned libspatialindex branch master.
2016-05-13 18:34:57,018 DEBUG  HEAD is now at 34fd22b Merge pull request #82 from miklos1/master

2016-05-13 18:34:57,023 DEBUG  
2016-05-13 18:34:57,030 DEBUG  
2016-05-13 18:34:57,040 DEBUG  
  glibtoolize or libtoolize not found. Giving up!
```

Using Ubuntu 16.04.

May be related to #779?

**Comment by miklos1** [comment]
Probably related. We are working on a fix, in the meantime, installing autoconf, automake and libtool will probably just solve the issue.

**Comment by miklos1** [comment]
Can you verify if `libspatialindex-dist` fixes this problem?

**Comment by dorugeber** [comment]
I can confirm that installing `autoconf`, `automake` and `libtool` first fixes the issue.  I don't have time to verify your second suggestion, sorry.

**Comment by miklos1** [comment]
Hopefully fixed by #783.
